### PR TITLE
Flip card

### PR DESCRIPTION
**Pull request by lenzloesche**

### CODE REVIEW

**Review by sahedw — APPROVED** [review]
lgtm

File: components/card/card.css [hunk 1]
```diff
@@ -46,3 +46,20 @@
...
+.transform-card {
+  transition: 0.2s ease-out;
+  transform: rotateY(360deg);
+}
+
+.transform-card-back {
+  transition: 0.3s ease-out;
+  transform: rotateY(0deg);
+}
+.card--front-side {
```
Comment (nikokreissl) [inline components/card/card.css]: Not sure but is this supposed to be here?

**Review by nikokreissl — APPROVED** [review]
lgtm and thanks for removing the commented lines 😊

File: components/card/card.css (hunk 1 above)
Comment (lenzloesche) [reply components/card/card.css]: no, i will fix it.

**Pull request merged**